Ingress and Terminating Gateway docs (#7710)
hashicorp/consul: website/pages/docs/agent/config-entries/terminating-gateway.mdx

---
layout: docs
page_title: 'Configuration Entry Kind: Terminating Gateway'
sidebar_title: terminating-gateway
description: >-

>   The terminating-gateway config entry kind allows for configuring terminating gateways

Reviewer: ```suggestion
  The `terminating-gateway` config entry kind allows for configuring terminating gateways
```

  to proxy traffic from services in the Consul service mesh to services outside the mesh.
---

# Terminating Gateway

-> **1.8.0+:** This config entry is available in Consul versions 1.8.0 and newer.

>   The `terminating-gateway` config entry kind allows for configuring terminating gateways

Reviewer: ```suggestion
  The `terminating-gateway` config entry kind allows you to configure terminating gateways
```

>   to proxy traffic from services in the mesh to services registered with Consul that do not have a

Reviewer: ```suggestion
  to proxy traffic from services in the Consul service mesh to services registered with Consul that do not have a
```

>   [Connect proxy](/docs/connect/proxies). See [Terminating Gateway](/docs/connect/terminating_gateway)

Reviewer: ```suggestion
  [Connect service sidecar proxy](/docs/connect/proxies). See [Terminating Gateway](/docs/connect/terminating_gateway)
```

  for more information.

## Wildcard service specification

>   Terminating gateways can optionally target all services within a namespace by specifying a wildcard "*"

Reviewer: See comment about about specifiy Consul namespace.
